npm/extensions.bzl

```diff
@@ -133,6 +134,8 @@ WARNING: Cannot determine home directory in order to load home `.npmrc` file in
         npm_auth = npm_auth,
     )
 
+    is_gnu_tar = str(check_is_gnu_tar(module_ctx))
```

Comment:
Can we do the `str()` in the `npm_import` macro instead of here? So the macro is `True|False|None` which converts it to the underling `"True"|"False"|""`...?

Reply:
that might be slightly less efficient, but sure

Reply:
Just because it does the conversion essentially at repo-rule runtime (N times) instead of repo-rule generation time (~1 time)?

Reply:
yep. probably doesn't matter

Reply:
It's just so much prettier 😅